Lib/test/test_pyrepl/test_pyrepl.py

```
@@ -44,14 +44,15 @@ def run_repl(
         *,
         cmdline_args: list[str] | None = None,
         cwd: str | None = None,
+        skip: bool = True,
```

Is it safe to make this `True` by default? There are other tests that currently call `run_repl` that do not check the output for `can't use pyrepl`. If for whatever reason the output ends up being `can't use pyrepl` they will be silently skipped as well. I'm not sure how much of a concern this is though..

---

Ok, fair point, I guess it is better to fail tests then skip by default. Will change once GitHub will be up again.